### ISSUE TITLE
Chat cards

### ISSUE THREAD

**Issue by zeel01** [zeel01/monarch#22]
Rich chat cards with information about cards that have been drawn, played, whatever.

**Comment by Tommycore** [comment]
Yes please. Especially when cards are **openly** played from one stack to another, i.e. hand to discard pile or something like that, it would make sense to have the chat card say *which* card it is.

**Comment by tmtabor** [comment]
This is the feature I want to see the most!
